data-request-reward-collateral-ratio.md

```diff
@@ -0,0 +1,122 @@
+
+At the moment, despite the average reward to collateral ratio being quite low, revealing an out-of-consensus value is not actually punishing because slashed collateral is distributed amongst honest nodes. While a node operator will lose significantly more collateral than the potential reward when he involuntarily reveals an out-of-consensus value, the fact that he can earn the slashed collateral of other operators when he is honest offsets this.
+
+However, redistributing slashed collateral introduces certain types of collateral-stealing attacks and [WIP-0023](https://github.com/drcpu-github/WIPs/blob/burn-slashed-collateral/burn-slashed-collateral.md) proposes to burn slashed collateral to mitigate these attacks. Under the assumption that this WIP is accepted, solving data requests with significantly imbalanced ratios does not make economic sense anymore. The only reason node operators should participate in solving them is because it improves their odds to mine blocks and earn the associated reward. This could result in node operators aiming to solve the minimum amount of data requests possible to still profit from increased mining odds which is obviously bad for the overall network health.
+
+A potential downside of introducing a reward to collateral ratio at the node level is that it is possible to manipulate the reputation system by creating and solving data requests which other nodes will not solve. This proposal solves this issue by introducing this ratio at the protocol level. A data request which does not adhere to the minimum reward to collateral ratio will be considered as invalid. If a block is proposed including such a data request, it is also considered as invalid.
+
+One additional advantage of a protocol-enforced reward to collateral ratio is that several (griefing) attacks becoming significantly less interesting from an economic viewpoint. Without a reward to collateral ratio, tt's possible to create data requests that lock up collateral from other nodes or aim to introduce collateral slashing for virtually no money. For example, you could create malicious requests requiring 100 WIT collateral and set all rewards to 1 nanoWIT resulting in the data request being virtually free. If a reward to collateral ratio is introduced these type of griefing attacks will cost a tangible amount of money to execute.
+
+## Specification
+
+### Introduction of a protocol-enforced data request reward collateral ratio
+
+Properly enforcing a reward to collateral ratio at the protocol level requires adding this ratio as a consensus constant. Adding a consensus constant will change the magic number nodes use to validate inter-node communication. Therefore, proper precautions need to be implemented such that this number only changes when the WIP is accepted.
+
+The moment the WIP is accepted, nodes which receive a data request that does not adhere to the reward to collateral ratio should refuse to add it to their local mempool.
```

Comment:
From a TAPI perspective, it's clearer to talk about _activation_ vs. _acceptance_.
```suggestion
The moment the WIP is activated, nodes which receive a data request that does not adhere to the reward to collateral ratio MUST refuse to add it to their local mempool.
```